### DISCUSSION TITLE
Experience-primer-copilot HELP

### DISCUSSION THREAD

**Discussion by IsaacMirandaCamargos** [community/community#102554] [GitHub Education]
### Hi everyone,

Hello, i am student how to use GitHub copilot by the GitHub classroom and i am block in the step 3 because there 'isn't Complementation Panel in my copilot.

Tutorial pictute:

![image](https://github.com/community/community/assets/88542223/d70955a5-b0e9-44b8-a016-8883bcef47a0)

My workspace in github:

![image](https://github.com/community/community/assets/88542223/7e9e3857-0575-40b8-a00c-079120f7100e)

Someone can help me?

**Comment by marcopros** [comment]
Press `ctrl + enter` to open the complementation panel. It worked for me.
